packages/gestalt/src/Masonry/defaultTwoColumnModuleLayout.js

```diff
@@ -402,15 +411,17 @@ const defaultTwoColumnModuleLayout = <T: { +[string]: mixed }>({
       }
 
       // For each unpainted item, start generating possible layouts
-      oneColumnItems.forEach((item, i, arr) => {
-        addPossibleLayout({
-          item,
-          i,
-          arr,
-          heightsArr: paintedItemHeights,
-          prevNode: startNodeData,
+      if (!skipGraph) {
```

Comment:
so just clarifying, we're still creating the graph with what's been painted so far but just not adding any layouts?

Reply:
Yeah, we create the graph and add the first node that is no layout. This is the one used as winning node and that puts the tcm right after painted items.